resources/views/campaigns/recovery/index.blade.php

```diff
@@ -9,51 +9,29 @@
 ])
 
 @section('content')
-
     <div class="flex gap-5 flex-col">
         @include('partials.errors')
         <div class="flex gap-2 items-center">
             <h3 class="inline-block grow">
-                @if (isset($isPost))
-                    {{ __('campaigns/recovery.post-title') }}
-                @else
-                    {{ __('campaigns/recovery.title') }}
-                @endif
+                {{ __('campaigns/recovery.title') }}
             </h3>
             <button class="btn2 btn-sm btn-ghost" data-toggle="dialog"
                     data-target="recovery-help">
                 <x-icon class="question" />
                 {{ __('crud.actions.help') }}
             </button>
-            @if(isset($isPost))
-                <a class="btn2 btn-sm" href="{{ route('recovery', [$campaign]) }}">
-                    <x-icon class="fa-solid fa-rotate" />
-                    {{ __('campaigns/recovery.toggle.entity') }}
-                </a>
-            @else
-                <a class="btn2 btn-sm" href="{{ route('recovery.posts', [$campaign]) }}">
-                    <x-icon class="fa-solid fa-rotate" />
-                    {{ __('campaigns/recovery.toggle.post') }}
-                </a>
-            @endif
         </div>
         @if (session()->get('boosted-pitch'))
```

Comment:
It's no longer possible to trigger this if, so it can be removed (and removed from the controller?)